proto/gds/models/v1/models.proto

```diff
@@ -50,6 +50,28 @@ message GDSExtraData {
     // Temporary: verification token for light weight authentication for verification
     // TODO: replace with admin API that uses authentication
     string admin_verification_token = 1;
+
+    // Change log which represents previous verification state changes made to a VASP
+    repeated VerificationChangeLogEntry verification_change_log = 2;
+}
+
+// VerificationChangeLogEntry contains information about a single verification state
+// change for a VASP.
+message VerificationChangeLogEntry {
+    // RFC3339 timestamp
+    string timestamp = 1;
+
+    // Previous verification state (handled internally) and current verification state
+    uint32 previous_state = 2;
+    uint32 current_state = 3;
```

Comment:
Storing these states as a `uint32` would certainly work, but would require additional parsing by the application. I recommend we simply store these as a `trisa.gds.models.v1beta1.VerificationState`; at the top of this protobuf file add an import to `import "trisa/gds/models/v1beta1/models.proto";` -- the `go generate` [command](https://github.com/trisacrypto/directory/blob/main/pkg/gds/models/v1/generate.go#L3) appears already setup to include the TRISA files. 

See https://github.com/trisacrypto/directory/blob/main/proto/gds/admin/v1/admin.proto for an example of the use of an imported VerificationState

Reply:
Thanks, I like the idea of using the actual VerificationState.